v2/nodejs/session/getSessionInformation.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: getSessionInformation
+title: getSessionInformation
+hide_title: true
+---
+
+## ``getSessionInformation(sessionHandle)``
+
+### Parameters
+- ``sessionHandle``
+  - type : ``string``
+
+### Returns
+- ``Promise<SessionInformation>`` returns all session related information for the given ``sessionHandle``.
```

Comment:
FYI: In line code can be done with one back tick.. you don't need two of them

Reply:
Yep, I just used the same style used in other docs in the folder. I can change it though